Userland/Libraries/LibRegex/RegexParser.cpp

```diff
@@ -1153,6 +1153,9 @@ bool ECMA262Parser::parse_quantifier(ByteCode& stack, size_t& match_length_minim
         auto low_bound = low_bound_string.to_uint();
 
         if (!low_bound.has_value()) {
+            if (done())
+                return set_error(Error::MismatchingBrace);
```

Comment:
This isn't an error, it just becomes a literal.
For example:
![Screenshot from 2021-07-24 10-49-56](https://user-images.githubusercontent.com/25595356/126864801-1bb1361a-6b63-4c87-b228-4d48f9cdf34c.png)

Reply:
Oh.
Then the previous should've been correct, hmmmm

Reply:
Looks like that's actually an annex B thing, the base ECMA262 grammar does not allow that.
See https://tc39.es/ecma262/#prod-annexB-ExtendedPatternCharacter vs. https://tc39.es/ecma262/#prod-PatternCharacter

So the error is correct, but only if we're not using the browser extended grammar.

Reply:
Updated to allow it in annex B mode.